### DISCUSSION TITLE
demo_client create socket failed, errno: 97

### DISCUSSION THREAD

**Discussion by SUOMALILI** [alibaba/xquic#93] [Q&A] [answered]
![Screenshot_20220216_213010](https://user-images.githubusercontent.com/47624845/154274758-7bce6d4d-0642-400b-b3ab-3e743dda3f04.png)
the build of quic is sucess and demo_server work, but demo_client create socket failed.

**Comment by ruiqizhou** [comment, marked as answer]
For testing the library, see the [Testing docs](https://github.com/alibaba/xquic/blob/main/docs/docs-zh/Testing-zh.md).

**Comment by vEuphemiav** [comment]
Have you solved this problem? How was it resolved?
